Entities/Industry/CTFShops/Quarry/Quarry.as

```diff
@@ -195,6 +204,13 @@ void spawnOre(CBlob@ this)
 	_ore.server_SetQuantity(!rare ? amountToSpawn : rare_output);
 
 	this.set_s16(fuel_prop, blobCount - actual_input); //burn wood
+	const string current_output = "current_quarry_output_" + this.getTeamNum();
+	
+	// reduce output if it's higher than minimal output
+	if (sv_gamemode == "CTF" || sv_gamemode == "SmallCTF")
```

Comment:
Hardcoding gamemodes like this is bad. Find a better way to only apply it to specific gamemodes.